src/sds.c

```diff
@@ -227,12 +227,12 @@ void sdsclear(sds s) {
 /* Enlarge the free space at the end of the sds string so that the caller
  * is sure that after calling this function can overwrite up to addlen
  * bytes after the end of the string, plus one more byte for nul term.
- * When exact is 0, enlarge to twice the needed size.
- * When exact is 1, enlarge fixed space of length 'addlen'.
+ * When greedy is 1, enlarge more than needed.
+ * When greedy is 0, enlarge fixed space of length 'addlen'.
```

Comment:
```suggestion
 * If there's already sufficient free space, this function returns without any
 * action, if there isn't sufficient free space, it'll allocate what's missing,
 * and possibly more:
 * When greedy is 1, enlarge more than needed, to avoid need for future reallocs
 * on incremental growth.
 * When greedy is 0, enlarge just enough so that there's free space for 'addlen'.
```